server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/event/ComputeMessagePreviewListener.java

```diff
@@ -101,7 +99,8 @@ private void handleAddedEvent(Added addedEvent, MailboxSession session) throws M
 
     private void preComputePreview(List<MessageId> messageIds, MailboxSession session) throws MailboxException {
         Flux.fromIterable(messageIdManager.getMessages(messageIds, FetchGroup.BODY_CONTENT, session))
-            .map(Throwing.function((MessageResult messageResult) -> computePreview(messageResult)))
+            .publishOn(Schedulers.boundedElastic())
+            .map(Throwing.function(this::computePreview))
```

Comment:
sneakyThrow ?

Reply:
We don't care here about the exception, it will go to dead letters anyway, where we can retry it.